### PR TITLE
DarkModeToggle: don't set schedule when updating from accountsservice

### LINKED ISSUE #8: Toggling dark mode disables schedule
### What Happened?

When clicking the dark mode toggle, any scheduled dark mode becomes disabled

### Steps to Reproduce

1. Have dark mode scheduled
2. Click the toggle
3. See that the schedule has been disabled in System Settings

### Expected Behavior

I should be able to toggle dark mode on and off without disrupting the schedule. Toggling dark mode on early in my day should still schedule back to light mode at sunrise. Toggling light mode at night should still schedule back to dark mode next sunset

Related: https://github.com/elementary/settings-daemon/issues/14

### OS Version

7.x (Horus)

### Software Version

Latest release (I have run all updates)

### Log Output

_No response_

### Hardware Info

_No response_

### PR DESCRIPTION
**Pull request by danirabbit**

### CODE REVIEW

File: src/Widgets/DarkModeToggle.vala [hunk 1]
```diff
@@ -19,23 +19,26 @@ public class QuickSettings.DarkModeToggle: SettingsToggle {
...
-            }
-        });
+        notify["active"].connect (on_activate);
 
         ((DBusProxy) pantheon_service).g_properties_changed.connect ((changed, invalid) => {
             var color_scheme = changed.lookup_value ("PrefersColorScheme", new VariantType ("i"));
             if (color_scheme != null) {
+                notify.disconnect (on_activate);
                 active = (Granite.Settings.ColorScheme) color_scheme.get_int32 () == Granite.Settings.ColorScheme.DARK;
+                notify["active"].connect (on_activate);
```
Comment (ryonakano) [inline src/Widgets/DarkModeToggle.vala:29]: I think we should leave some comments why disconnecting and re-connecting here.

**Comment by ryonakano** [comment]
Hmm, I'm still facing the linked issue with this branch, even after installing it and rebooting…

**Comment by danirabbit** [comment]
@ryonakano oops sorry you're right. This doesn't fix #8. The issue this fixes instead is:

1. Open System Settings → Desktop → Appearance and select either Default at night or Dark during the day
2. Select "Sunset to sunrise" and see that the style changes from light to dark (or dark to light)

Without this branch:
3. Schedule gets reset to "Disabled" after the style changes

With this branch:
4. Schedule remains on "sunset to sunrise"

**Review by zeebok — APPROVED** [review]
This does fix the schedule reset you explained

**Pull request merged**